Tools/DumpRenderTree/TestOptions.cpp

```diff
@@ -150,7 +150,7 @@ const TestFeatures& TestOptions::defaults()
             { "JavaScriptEnabled", true },
             { "KeygenElementEnabled", false },
             { "LayoutFormattingContextIntegrationEnabled", true },
-            { "LineHeightUnitsEnabled", false },
+            { "LineHeightUnitsEnabled", true },
```

Comment:
I think this can be removed if UnifiedWebPreferences.yaml sets the status to stable (which was the intent of the other commit)

Reply:
Okay, I'll update the patch to do that.

I'll also modify any layout tests which flip the flag individually.